ztag/schema.py

```diff
@@ -186,25 +201,53 @@ class CensysString(WhitespaceAnalyzedString):
         }),
         "subject_alt_name":alternate_name,
         "issuer_alt_name":alternate_name,
-        "crl_distribution_points":ListOf(URI()),
+        "crl_distribution_points":ListOf(URL()),
         "authority_key_id":HexString(),
         "subject_key_id":HexString(),
-        "extended_key_usage":ListOf(Signed32BitInteger()), # TODO: what sized integer should this be?
-        "certificate_policies":ListOf(CensysString()),
+        "extended_key_usage":SubRecord({
+            "value":ListOf(Signed32BitInteger()), # TODO: what sized integer should this be?
```

Comment:
There's only 9 defined in the RFC, but Golang parses them as a 32-bit signed integer.